server.js

```diff
@@ -96,8 +96,11 @@ app.prepare().then(() => {
       })
       res.end()
     } else if (/^\/doc.*/i.test(pathname)) {
-      // path /doc*/... -> /doc/...
-      let normalized_pathname = pathname.replace(/^\/doc[^?/]*/i, '/doc')
+      // path /doc*/{item} -> /doc/{item} and trim trailing / if present
```

Comment:
why would we want to keep `doc-something`? can we do exclusively `docs` and `documentation`? - that should be enough.

Reply:
and let's create a ticker for a proper normalization for all URLs? there should be a simple general way to achieve this.

Reply:
Idk. This is the way the redirect was before. I thought whoever wrote this meant to catch all "document"-like words. I can limit it to docs and documentation.

Reply:
Fixed but may need double checking... Will open a review.

Reply:
there was not specific reason behind this decision. I think it's outdated and we can make it more strict and aligned with what we actually need on the dvc.org . As I mentioned - I can understand `docs` or `documentation` being aliases for the `doc`. But we should stop supporting some random URLs - I don't see any reasons for this.

Reply:
OK. I'll follow up in https://github.com/iterative/dvc.org/pull/882#pullrequestreview-337006182